### DISCUSSION TITLE
Translations || which branch and release to follow?

### DISCUSSION THREAD

**Discussion by przemys1aw** [flameshot-org/flameshot#2309] [Q&A] [answered]
Hi there,

can you hint me which branch and which _Flameshot_ release to follow should I want to update files in the _data/translations/_ directory?

And generally can you point me to some guidelines describing your commit habits?

kind regards,
p

**Comment by borgmanJeremy** [comment, marked as answer]
Hello,

To make code contributions please submit a PR against the master branch. We merge PR's to master and periodically freeze the master branch for a release. 

If you specifically want to update the translations please submit that through our weblate project.
